DQM/L1TMonitor/python/L1TCSCTF_cfi.py

```diff
@@ -17,5 +17,6 @@
 #
 # Make changes for running in Run 2
 #
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
+# This is not working.
+#from Configuration.StandardSequences.Eras import eras
+#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
```

Comment:
@davidlange6 
I think if we want to solve this, it will be better to explain a little bit more to each other .... 

For the recent configuration: 
============================================
l1tCsctf = cms.EDAnalyzer("L1TCSCTF",
    gmtProducer = cms.InputTag("gtDigis"),

    statusProducer = cms.InputTag("csctfDigis"),
    outputFile = cms.untracked.string(''),
    lctProducer = cms.InputTag("csctfDigis"),
    verbose = cms.untracked.bool(False),
    gangedME11a = cms.untracked.bool(True),
    trackProducer = cms.InputTag("csctfDigis"),
    mbProducer = cms.InputTag("csctfDigis:DT"),
    DQMStore = cms.untracked.bool(True),
    disableROOToutput = cms.untracked.bool(True)
)

#
# Make changes for running in Run 2
#
from Configuration.StandardSequences.Eras import eras
eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
============================================

the eras.run2_common.toModify( l1tCsctf, gangedME11a = False ) try to overwrite the flag gangedME11a to False. But I found is it doesn't work. I cout this flag in the code: 
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/DQM/L1TMonitor/src/L1TCSCTF.cc#L51
it is still True.... 

Can you explain why it is ...